src/index.css

```diff
@@ -0,0 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: rgb(188, 188, 188);
+}
+
+li {
+  list-style: none;
+  border-bottom: 1px solid rgb(211, 211, 211);
+  display: flex;
+  align-items: center;
+  padding-bottom: 10px;
+}
+
+.main-list {
+  background-color: #fff;
+  margin: 50px 20%;
+}
+
+.main-list > * {
+  padding: 10px;
+}
+
+.inputbox {
+  outline: none;
+  height: 35px;
+  flex-grow: 1;
+  border: none;
+  background-color: transparent;
+  resize: none;
+  font-size: 20px;
+  margin-left: 8px;
+  cursor: default;
+}
+
+.checkbox {
+  width: 30px;
+  height: 30px;
+}
+
+.rmv {
+  font-size: 45px;
+  cursor: pointer;
+  width: 60px;
```

Comment:
It will be good if you adopt using responsive units instead of fixed units as this will make your site more responsive.